packages/emulation/libretro-atari800/package.mk

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libretro-atari800"
+PKG_VERSION="70182481ec0ce2d027ed6fb3f7296d9662c7d58e"
+PKG_SHA256="12edac9c3a4cacdd1139f21d2cafee1d66152d53d1114ee2bf613dcb8b827bbe"
+PKG_LICENSE="GPLv2"
+PKG_SITE="https://github.com/libretro/libretro-atari800"
+PKG_URL="https://github.com/libretro/libretro-atari800/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain kodi-platform"
+PKG_LONGDESC="Atari 8-bit computer and 5200 console emulator"
+PKG_TOOLCHAIN="make"
+
+PKG_MAKE_OPTS_TARGET="platform=unix"
+
+PKG_LIBNAME="atari800_libretro.so"
+PKG_LIBPATH="$PKG_LIBNAME"
+PKG_LIBVAR="ATARI800_LIB"
+
+makeinstall_target() {
+  mkdir -p $SYSROOT_PREFIX/usr/lib/cmake/$PKG_NAME
+  cp $PKG_LIBPATH $SYSROOT_PREFIX/usr/lib/$PKG_LIBNAME
+  echo "set($PKG_LIBVAR $SYSROOT_PREFIX/usr/lib/$PKG_LIBNAME)" > $SYSROOT_PREFIX/usr/lib/cmake/$PKG_NAME/$PKG_NAME-config.cmake
+}
+
```

Comment:
extra newline or just strange github formating ?

Reply:
newline dropped (also in backport)